common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -149,15 +147,22 @@ private String getBaseName(String name) {
         return bracketIndex == -1 ? name : name.substring(0, bracketIndex);
     }
 
-    private boolean similarStack(ItemStack firstItem, ItemStack secondItem) {
-        if (!firstItem.getItem().equals(secondItem.getItem())) return false;
+    private boolean sameGearWithSkin(ItemStack existingItem, ItemStack newItem) {
+        if (WynnItemMatchers.isGearSkin(existingItem) || !WynnItemMatchers.isGearSkin(newItem)) return false;
```

Comment:
I've tried hard to keep "specific" item knowledge out of the ItemHandler. Before you merge this, let me check a bit closer if there is a way to solve this which does not mean that the ItemHandler has to know about gear and skins...